input-explainer.md

```diff
@@ -330,29 +330,29 @@ The UA may update the `gamepad` state at any point, but it must remain constant
 
 ### XR gamepad mapping
 
-The WebXR Device API also introduces a new standard controller layout indicated by the `mapping` value of `xr-standard`. (Additional mapping variants may be added in the future if necessary.) This defines a specific layout for the inputs most commonly found on XR controller devices today. The following table describes the buttons/axes and their physical locations:
+The WebXR Device API also introduces a new standard controller layout indicated by the `mapping` value of `xr-standard`. (Additional mapping variants may be added in the future if necessary.) This defines a specific layout for the inputs most commonly found on XR controller devices today. The following table describes the buttons/axes and their associated physical inputs:
 
-| Button     | `xr-standard` Location            |
-| ---------- | --------------------------------- |
-| buttons[0] | Primary trigger                   |
-| buttons[1] | Primary Touchpad/Joystick click   |
-| buttons[2] | Grip/Secondary trigger            |
-| buttons[3] | Secondary Touchpad/Joystick click |
+| Button     | `xr-standard` Binding  |
+| ---------- | -----------------------|
+| buttons[0] | Primary trigger        |
+| buttons[1] | Touchpad press         |
```

Comment:
Nit: Should we group the primary/secondary triggers together as 0/1 and touchpad/thumbstick together as 2/3 in the button order?  I believe the reason they were split up like this before was to keep the primary inputs before the secondary inputs in the list.  Does this ordering still imply that touchpad is more "primary" than thumbstick?

Reply:
I wondered about that too. I'm not opposed to it. Would probably help it "read" a bit better. I think I'll make that change.

As for touchpads being "more primary" than thumbsticks, yeah, it kinda does? I don't think it's ultimately going to matter much unless you've got developers who are just blindly responding to axes[0] and [1] and nothing else, which is a recipe for failure on their part right off the bat. I did do a quick inventory of the input devices I could think of off the top of my head and by a fair margin (9-5) touchpads are more common than thumbsticks, so I felt that ordering was justified.

Reply:
Yea - I think it makes sense to still keep touchpad (`[2]`) before thumbstick (`[3]`).  Just moving generally away from the notion that this is a sorted list of buttons by "primariness" seems worthwhile.